appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/util/ConnectionPoolObjectsUtils.java

```diff
@@ -330,12 +332,10 @@ public static Subject createSubject(ManagedConnectionFactory mcf,
                         new PasswordCredential(prin.getName(),
                                 password.toCharArray());
                 pc.setManagedConnectionFactory(mcf);
-                AccessController.doPrivileged(new PrivilegedAction() {
-                    public Object run() {
-                        tempSubject.getPrincipals().add(prin);
-                        tempSubject.getPrivateCredentials().add(pc);
-                        return null;
-                    }
+                AccessController.doPrivileged((PrivilegedAction<Object>) () -> {
```

Comment:
dtto - action to the variable. Also better use type Void instead of Object:
```
PrivilegedAction<Void> action = () -> { xxx ... return null;};
AccessController.doPrivileged(action);
```